### PR TITLE
Added transform

### LINKED ISSUE #21: Write transform script
## Description
Write `transform.py` for cleaning and processing data collected from extract

---

## Outcome
Cleans data from extract into an input compatible with loading into the database for `load` stage

---
## User Story
As a developer, we need to clean data to ensure that there are no erroneous inputs and that any unwanted datapoints are handled. This ensures all data being uploaded to the database is in the desired format and can be easily handled.

### PR DESCRIPTION
**Pull request by Jakub-Poskrop**
Closes #21 - added transform, let me know what I may have missed or should add.

### CODE REVIEW

**Review by Fahi28 — CHANGES_REQUESTED** [review]
make sure all the tests and pylint checks pass

File: pipeline/transform.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+'''
+Module that transforms the extracted data into the necessary format. 
+Ready to be loaded into the database
+'''
+import pandas as pd
+import datetime
+
+
+def clean_data(earthquake_data: list[dict]) -> pd.DataFrame:
```
Comment (S1mpySloth) [inline pipeline/transform.py:9]: Docstring perhaps?

File: pipeline/transform.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+'''
+Module that transforms the extracted data into the necessary format. 
+Ready to be loaded into the database
+'''
+import pandas as pd
```
Comment (S1mpySloth) [inline pipeline/transform.py:5]: import datetime before pandas?

**Review by qrafiq1 — APPROVED** [review]
Looks good!

**Review by Fahi28 — APPROVED** [review]
much better

**Pull request merged**